pkg/controllers/allocation/controller.go

```diff
@@ -23,17 +23,31 @@ import (
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
 	"github.com/awslabs/karpenter/pkg/packing"
 	"github.com/awslabs/karpenter/pkg/utils/apiobject"
+	podutils "github.com/awslabs/karpenter/pkg/utils/pod"
 
 	"go.uber.org/zap"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	MaxBatchWindow   = 10 * time.Second
```

Comment:
Thoughts on putting these fields onto the provisioner?

Reply:
At first, I was thinking that would be a great idea to do. However, since we have two fields to adjust, I think we might be able to configure a pretty good default. Either way, I think I'd opt for "not yet" and see if it's needed.

Reply:
Sounds good.